### DISCUSSION TITLE
Add Github sponsor to repository

### DISCUSSION THREAD

**Discussion by Putr** [iSoron/uhabits#606] [General Discussion]
Hi, love the app!

The app works great, is opensource and as I can see has some great ideas and leadership for future development.

I would be more than willing to a few coffees per month into supporting this project. So, @iSoron enable the feature, you deserve the cash. Even if it's probably not gonna be that much.

**Comment by ramnes** [comment]
I usually don't bump old issues / discussions because of the noise, but I'd love to support you through GitHub sponsors as well @iSoron. Is there anything in particular holding you back from adding the button on your profile? :)
